README.md

```diff
@@ -0,0 +1,13 @@
+# dat_pyinthesky
+
+This repository is a sandbox and drafting space for assorted efforts for STScI's data analysis tools. (Particularly, but not exclusively, for the Data Analysis Tools Branch.)
+
+## If you locally cloned this repo before 5 Feb 2021
+
+The primary branch for this repo has been transitioned from ``master`` to ``main``.  If you have a local clone of this repository and want to keep your local branch in sync with this repo, you'll need to do the following:
+```
+% git branch -m master main
+% git fetch origin
+% git branch -u origin/main main
+```
```

Comment:
A minor comment is that the "%" might confuse some people. Believe it or not, I have gotten help desk questions related to "bash$" in the instructions.

Reply:
Oh good point.  Maybe I'll just remove that and add something like "in your terminal".